.eslintignore

```diff
@@ -2,5 +2,8 @@ build/**
 dist/*
 **/vendor/**/*.js
 
+# TODO - Lint these files
+rollup*.config.js
```

Comment:
ESLint gets confused by the fact that Rollup config files are ES modules. It isn't super-important to lint these, but we can also probably fix this by telling ESLint to assume all files in the repo are ES modules by default.